### PR TITLE
add(platform): download a BYOC infra template

### PR DESCRIPTION
**Pull request by wojcik-dorota**
https://aiven.atlassian.net/browse/MILK-155

## Checklist

- [x] The first paragraph of the page is on one line.
- [x] The other lines have a line break at 90 characters.
- [x] I checked the output.
- [x] I applied the [style guide](../styleguide.md).
- [x] My links start with `/docs/`.

### CODE REVIEW

**Review by jeremy-heller-aiven — APPROVED** [review]
Looks good to me!

File: docs/platform/howto/byoc/download-infrastructure-template.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    [custom cloud created](/docs/platform/howto/byoc/create-custom-cloud) in your Aiven organization
+-   Access to the [Aiven Console](https://console.aiven.io/) or the
+    [Aiven CLI](/docs/tools/cli) installed
+
+## Download the template via console
+
+1.  Log in to the [Aiven Console](https://console.aiven.io/), and go to your organization.
+1.  Click **Admin** in the top navigation, and click <ConsoleLabel name="bringyourowncloud"/>
+    in the sidebar.
+1.  Select a desired cloud, and find **Infrastructure template** on the selected cloud's
```
Comment (staceysalamon-aiven) [inline docs/platform/howto/byoc/download-infrastructure-template.md:24]: A suggestion to reduce wordiness.

```suggestion
1.  Select a cloud, and find the **Infrastructure template** on the 
```

File: docs/platform/howto/byoc/download-infrastructure-template.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+---
+title: Download a custom cloud infrastructure template
+sidebar_label: Download infra template
+eywords: [Terraform, deployment, deploy, byoc, bring your own cloud, custom cloud]
```
Comment (staceysalamon-aiven) [inline docs/platform/howto/byoc/download-infrastructure-template.md:4]:
```suggestion
keywords: [Terraform, deployment, deploy, byoc, bring your own cloud, custom cloud]
```

**Review by staceysalamon-aiven — APPROVED** [review]
Looks fine to me. I only made one minor suggestion on wording and fixed a typo.

**Pull request merged**